src/main/java/info/novatec/camunda/migrator/ProcessInstanceMigrator.java

```diff
@@ -32,6 +32,7 @@
     public void migrateInstancesOfAllProcesses() {
         processEngine.getRepositoryService().createProcessDefinitionQuery()
             .active()
+            .latestVersion()
```

Comment:
Don't you want to add a test for this?

Reply:
I´m not sure how I could test this, since it only influences the number of times the method "migrateProcessInstances" is called.